queue_services/entity-emailer/src/entity_emailer/email_templates/BC-BN.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <base href="/">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="referrer" content="origin-when-cross-origin"/>
+    <meta name="author" content="BC Registries and Online Services">
+    {% if title %}
+    <title>{{ title }}</title>
+    {% else %}
+    <title>{{business.legalName}} - Business Number Information</title>
```

Comment:
In 2 of the other email templates, I removed the conditional here and just hard-coded the title.
1. just remove the conditional here (because I think we never set the 'title' variable) and leave line 12
2. please update BC-MRAS.html similarly

Reply:
(The conditional title only made sense if this template file was used for different types of emails... but it isn't so let's keep it clean and simple.)

The only downside is the email processor sets the email subject, and the templates set the document title, to the same values... This doesn't really bother me :)